scripts/us_cdc/500_places/README.md

```diff
@@ -0,0 +1,94 @@
+
+### Notes and Caveats
+
+None.
+
+### License
+The data is made available for public-use by the [CDC](https://www.cdc.gov/nchs/data_access/ftp_data.htm). Users of CDC National Center for Health Statistics Data must comply with the CDC's [data use agreement](https://www.cdc.gov/nchs/data_access/restrictions.htm).
+
+### Dataset Documentation and Relevant Links
+These data were collected and provided by the [CDC National Center for Chronic Disease Prevention and Health Promotion](https://www.cdc.gov/chronicdisease/index.htm). The documentation for the datasets is accessible [here](https://www.cdc.gov/places/about/index.html).
+
+## About the Import
+
+### Artifacts
+
+#### Scripts
+[`parse_cdc_places.py`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/parse_cdc_places.py)
+
+#### Test Scripts
+[`parse_cdc_places_test.py`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/parse_cdc_places_test.py)
+
+#### Test Files
+[`sample_county.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_county.csv)
+
+[`sample_county_expected.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_county_expected.csv)
+
+[`sample_city.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_city.csv)
+
+[`sample_city_expected.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_city_expected.csv)
+
+[`sample_census_tract.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_census_tract.csv)
+
+[`sample_census_tract_expected.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_census_tract_expected.csv)
+
+[`sample_zip_code.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_zip_code.csv)
+
+[`sample_zip_code_expected.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_zip_code_expected.csv)
+
+#### tMCFs
+[`cdc_places.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/cdc_places.tmcf)
+
+### Import Procedure
+
+#### Testing
+
+##### Test Data Cleaning Script
+
+To test the data cleaning script, run:
+
+```bash
+$ python3 parse_cdc_places_test.py
+```
+
+The expected output of this test can be found in the [`test_data`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/) directory.
+
+#### Processing Steps
+
+`@input_file_name` - path to the input csv file to be cleaned
+
+`@output_file_name` - path to write the cleaned csv file
+
+To clean the data files, run:
+
+```bash
+$ python3 parse_cdc_places.py input_file_name output_file_name
+```
```

Comment:
Could we have a wrapper shell script or something that downloads the CSV files and invokes this script for every CSV?

The idea is to make it as straightforward as possible, so that someone redoing the import can just copy/paste the commands from here, and it should work.

If the CSVs aren't too big, you could have a shell/py script to download them. See prior examples in:
- https://github.com/datacommonsorg/data/tree/master/scripts/us_census/decennial#generation
- https://github.com/datacommonsorg/data/blob/master/scripts/us_eia/opendata/download_bulk.py

Reply:
done.

Reply:
Thanks for the download script.

Can we please fully list the commands to run, so one can just copy paste the command line?  I would just add the following snippet to a shell-script and suggest running that after the download script. 

```
#!/bin/bash

mkdir -p cleaned_csv
for in_file in county_raw_data.csv city_raw_data.csv censustract_raw_data.csv zipcode_raw_data.csv; do
  out_file=$(echo $in_file | sed 's/_raw_data/_cleaned/g')
  python3 parse_cdc_places.py raw_data/$in_file cleaned_csv/$out_file
done
```

Also, please pass the right delimiter for the file.  Again, the idea is to make it as automatable as possible.

Reply:
Done! The delimiter for all files is a comma, so specified that in the shell script.

Reply:
Thanks!